Makefile

```diff
@@ -71,7 +72,7 @@ checkformatting: python
 	@tox -qe checkformatting
 
 .PHONY: test
-test: backend-tests coverage frontend-tests
+test: backend-tests frontend-tests
```

Comment:
Changed `make test` so that it doesn't run `coverage`. You can run `make coverage` separately or you can run `make test coverage`. This is how it's done in other projects (e.g. bouncer, h)